src/bp/core/services/hook/hook-service.ts

```diff
@@ -17,7 +17,7 @@ import { VmRunner } from '../action/vm'
 import { Incident } from '../alerting-service'
 
 const debug = DEBUG('hooks')
-const SYNC_DEBOUNCE_DELAY = ms('2s')
+const DEBOUNCE_DELAY = ms('2s')
```

Comment:
I know its not part of this PR but it seems to me that the code here and in action service is pretty much duplicated